services/httpd/service.go

```diff
@@ -101,6 +109,26 @@ func (s *Service) Open() error {
 		s.ln = listener
 	}
 
+	// Open unix socket listener.
+	if s.unixSocket {
```

Comment:
I don't think Windows supports unix sockets. Can you check for if the OS is windows and return an error if somebody is trying to use unix sockets on Windows?